src/pages/documentation/media-for-edge-network/tabs/api-reference.md

```diff
@@ -0,0 +1,2117 @@
+      */
+      public static final String AOD = "aod";
+  }
+
+}
+```
+
+<Variant platform="ios" api="stream-type" repeat="6"/>
+
+```swift
+public class MediaConstants: NSObject {
+  @objc(AEPEdgeMediaStreamType)
+  public class StreamType: NSObject {
+     // Constant defining stream type for VOD streams.
+        public static let VOD = "vod"
+     // Constant defining stream type for Live streams.
+        public static let LIVE = "live"
+     // Constant defining stream type for Linear streams.
+        public static let LINEAR = "linear"
+     // Constant defining stream type for Podcast streams.
+        public static let PODCAST = "podcast"
+     // Constant defining stream type for Audiobook streams.
+        public static let AUDIOBOOK = "audiobook"
+     // Constant defining stream type for AOD streams.
+        public static let AOD = "aod"
+    }
+}
+```
+
+**Example**
+
+#### Swift
+
+```swift
+var mediaObject = Media.createMediaObjectWith(name: "video-name", 
+                                                id: "videoId", 
+                                            length: 60, 
+                                        streamType: MediaConstants.StreamType.VOD,    
+                                         mediaType: MediaType.Video)
+```
+
+#### Objective-C
+
+```objc
+NSDictionary *mediaObject = [AEPMobileEdgeMedia createMediaObjectWith:@"video-name" 
+                                                               id:@"video-id" 
+                                                           length:60
+                                                       streamType:AEPEdgeMediaStreamType.VOD      
+                                                        mediaType:AEPEdgeMediaTypeVideo];
+```
+
+<Variant platform="android" api="standard-video-constants" repeat="1"/>
+
+```java
+public class MediaConstants {
+
+  public static final class VideoMetadataKeys {
+      public static final String SHOW = "a.media.show";
+      public static final String SEASON = "a.media.season";
+      public static final String EPISODE = "a.media.episode";
+      public static final String ASSET_ID = "a.media.asset";
+      public static final String GENRE = "a.media.genre";
+      public static final String FIRST_AIR_DATE = "a.media.airDate";
+      public static final String FIRST_DIGITAL_DATE = "a.media.digitalDate";
+      public static final String RATING = "a.media.rating";
+      public static final String ORIGINATOR = "a.media.originator";
+      public static final String NETWORK = "a.media.network";
+      public static final String SHOW_TYPE = "a.media.type";
+      public static final String AD_LOAD = "a.media.adLoad";
+      public static final String MVPD = "a.media.pass.mvpd";
+      public static final String AUTHORIZED = "a.media.pass.auth";
+      public static final String DAY_PART = "a.media.dayPart";
+      public static final String FEED = "a.media.feed";
+      public static final String STREAM_FORMAT = "a.media.format";
+  }
```

Comment:
Can you double check these values? For Edge Media, I think we dropped the "a.media." prefix for each value.

Reply:
Yes we have the prefix